examples/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
-identity = { path = "../identity", features = ["account", "stronghold"] }
+identity = { path = "../identity", features = ["account", "stronghold", "encryption-decryption"] }
```

Comment:
This seems unnecessary, because the encryption API is enabled by default in `identity-account`.